Create pull_request_template.md (#156)
rsksmart/lumino: docs/.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

# Description

> ´Describe the objective of the PR and a brief description of the changes introduced´

Reviewer: ```suggestion
Describe the objective of the PR and add a brief description of the changes introduced.
```

> 

Reviewer: ```suggestion
```

# Resolves

> `Point here the jira issue`

Reviewer: ```suggestion
Add a reference to the JIRA issue here.
```

# Impact

> `Describe the impact of the PR, wich specifics components of Lumino it affects`

Reviewer: ```suggestion
Describe the impact of the PR and which specifics components of Lumino it affects.
```

# Changelog

> `A bullets list of changes`

Reviewer: ```suggestion
A bullet list of changes.
```

> ```
> 1. added foo bar
> 2. refactored Foo class
> 3. moved foo to bar folder
> ```

Reviewer: ```suggestion
1. add foo bar
2. refactor Foo class
3. move foo to bar folder
```

Reviewer: as a personal preference, i would opt for actions in infinitive

> # TODO's

Reviewer: ```suggestion
# TODOs
```

Reviewer: this isn't really an acronym, but because it is styled as such, we don't need to add an apostrophe when using the plural form

> `You can use this section as TODO list. Use this space for when the PR is on draft state. `

Reviewer: ```suggestion
You can use this section as TODO list. Use this space for when the PR is on draft state.
```

> ```
> 1. review FOO class 
> 2. fix pending bug about bar 
> 3. document foo bar workflow
> ```

Reviewer: ```suggestion
- [x] review FOO class 
- [ ] fix pending bug about bar 
- [ ] document foo bar workflow
```

Reviewer: i think a checklist here might be more useful for a TODO list
